OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -150,7 +150,10 @@ static int HashList<T>(List<T> xs)
 		List<CPos> EvalPath(BlockedByActor check)
 		{
 			var path = getPath(check).TakeWhile(a => a != mobile.ToCell).ToList();
-			mobile.PathHash = HashList(path);
+
+			if (Game.Settings.Debug.PathDebug)
```

Comment:
Desync alert!

This needs to be enabled for all players, or for none.

Can anyone remember when we last had a pathfinding related desync? Maybe just remove it completely, otherwise we can have this as a server-controlled option like we do sync reports.

Reply:
![Screenshot 2021-03-22 at 20 18 16](https://user-images.githubusercontent.com/167819/112053045-c5118580-8b4b-11eb-8c86-b892486fd7cc.png)

😅

Reply:
Heh, yeah. Only been 5 years or so 😅

Reply:
9!

Reply:
Oh wow, I though it was from the last pathfinding refactor...